readme.md

```diff
@@ -547,6 +547,7 @@
 - [Minecraft](https://github.com/bs-community/awesome-minecraft#readme) - Sandbox video game.
 - [Game Datasets](https://github.com/leomaurodesenv/game-datasets#readme) - Materials and datasets for Artificial Intelligence in games.
 - [Haxe Game Development](https://github.com/Dvergar/awesome-haxe-gamedev#readme) - A high-level strongly typed programming language used to produce cross-platform native code.
+- [Friday Night Funkin'](https://github.com/FunkinCrew/funkin-resources#readme) - A rhythm game made in HaxeFlixel.
```

Comment:
I would change this line to read the following:

```
- [Friday Night Funkin'](https://github.com/FunkinCrew/funkin-resources) - A cartoon rhythm game made for Newgrounds.
```

Reply:
According to the guidelines, the #readme is required which is also more user friendly to directly jump to it when opening the link.

Reply:
Ok so the correct description should be:

```
- [Friday Night Funkin'](https://github.com/FunkinCrew/funkin-resources#readme) - A cartoon rhythm game made for Newgrounds.
```